### PR TITLE
update kernel reqs doc; recommend updates on RHEL

### PR DESCRIPTION
**Pull request by SvenDowideit**
replaces  #9456

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@unclejack @fredlf @jamtur01

**Comment by unclejack** [comment]
LGTM

**Comment by thaJeztah** [comment]
Typing on mobile, so can't comment inline;

> Please keep your system is up to date using `yum update` and rebooting

s/is up to date/up to date/

**Comment by SvenDowideit** [comment]
@thaJeztah merci :) it takes a village!

**Comment by jamtur01** [comment]
LGTM

File: docs/sources/installation/binaries.md [hunk 1]
```diff
@@ -32,18 +32,52 @@ runtime:
...
+types of exploits. Your distribution's documentation should provide
+detailed steps on how to enable the recommended security mechanism.
+
+Some Linux distributions enable AppArmor or SELinux by default and
+they run a kernel which doesn't meet the minimum requirements (3.10
+or newer). Updating the kernel to 3.10 or newer on such a system
+might not be enough to start Docker and run containers.
+Incompatibilities between the version of AppArmor/SELinux user
+space utilities provided by the system and the kernel could prevent
+Docker from running, from starting containers or make containers
```
Comment (fredlf) [inline docs/sources/installation/binaries.md]: Comma after "starting containers" please.

**Comment by fredlf** [comment]
One small copy edit, otherwise LGTM!

**Pull request merged**